tests/test_inlinequery.py

```diff
@@ -68,6 +68,15 @@ def test(*args, **kwargs):
         monkeypatch.setattr(inline_query.bot, 'answer_inline_query', test)
         assert inline_query.answer()
 
+    def test_answer_auto_pagination(self, monkeypatch, inline_query):
+        def test(*args, **kwargs):
+            inline_query_id = args[0] == inline_query.id
```

Comment:
Might aswell make the assertions inside the callback directly. The `inline_query_id` naming doesn't imply that this is a boolean holding a test result. If you insist on returning it from the callback using `x and y`, then name it `inline_query_id_matches_expected` or so :)